package.json

```diff
@@ -27,36 +27,36 @@
     "ember-deep-set": "^0.1.3"
   },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
-    "ember-cli": "~3.0.0",
+    "broccoli-asset-rev": "^2.7.0",
+    "ember-cli": "~3.2.0",
     "ember-cli-dependency-checker": "^2.0.0",
     "ember-cli-eslint": "^4.2.1",
     "ember-cli-htmlbars": "^2.0.3",
     "ember-cli-htmlbars-inline-precompile": "^1.0.2",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-qunit": "^4.1.1",
+    "ember-cli-qunit": "^4.3.2",
     "ember-cli-release": "^1.0.0-beta.2",
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^2.0.0",
-    "ember-data": "^3.0.0",
+    "ember-data": "^3.2.0",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.0.0",
+    "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-native-dom-helpers": "^0.6.2",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~3.0.0",
+    "ember-source": "~3.2.0",
     "ember-source-channel-url": "^1.0.1",
     "ember-try": "^0.2.23",
     "eslint-plugin-ember": "^5.0.0",
-    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-node": "^6.0.1",
     "flow-bin": "^0.61.0",
     "flow-typed": "^2.2.3",
     "loader.js": "^4.2.3"
   },
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "6.* || 8.* || >= 10.*"
```

Comment:
In theory dropping node 4 support is a breaking change so we should probably leave that

Reply:
Since we are on a beta branch, do you think its ok to leave out?  I think node 4.* only supports up to 3.1 as well.  Lmk what you think.

https://github.com/ember-cli/ember-cli/pull/7801/files

Reply:
🤷‍♂️  I don't have a strong feeling about it since my experience is that I haven't used node 4 for a very long time, but I know I've seen other addons be cautious about this.

Reply:
@Dhaulagiri Yeah I think you are right.  I put the `^4.5` back 👍